common/check_internal.h

```diff
@@ -37,7 +37,7 @@ class ExitingStream {
   [[noreturn]] ~ExitingStream() {
     llvm_unreachable(
         "Exiting streams should only be constructed by check.h macros that "
-        "ensure the special operator| exits the program prior to their "
+        "ensure the pipe operator | exits the program prior to their "
```

Comment:
This feels a little redundant. How about this instead?

```suggestion
        "ensure the `|` operator exits the program prior to their "
```

Reply:
Yeah that also works.